components/ws-manager-bridge/src/meta-instance-controller.ts

```diff
@@ -34,10 +34,15 @@ export class MetaInstanceController {
                 const now = Date.now();
                 const creationTime = new Date(instance.latestInstance.creationTime).getTime();
                 const timedOutInPreparing = now >= creationTime + (this.config.timeouts.preparingPhaseSeconds * 1000);
+                const timedOutInStopping = now >= creationTime + (this.config.timeouts.stoppingPhaseSeconds * 1000);
```

Comment:
Shouldn't the time difference be calculated based on `stoppingTime` instead of `creationTime`?

E.g. if you start a workspace, then work for 2 hours, and then stop it -- that workspace is at risk of a false-positive force-stop.

I believe the same applies to `unknown` (any workspace however briefly in `unknown` is at risk of force-stop if it was created over 10 minutes ago -- except if `unknown` can only happen on start-up, I'm not 100% sure about that.) `preparing` seems safe as it is.

Reply:
The preparing and unknown was done in collaboration with @csweichel so my guess is that the `unknown` is ok.

I'll make the change the `stoppingTime` as your point seems logical to me

Reply:
@jankeromnes the `stoppingTime` is a bit fiddly as it's only an optional param (presumably, only set if it's entered `stopping` phase). I've defaulted it to the `now` time, which makes it more complex than I'd ideally like, but its the simplest way I could think of doing it - it shouldn't add any false positives as the time elapsed should be `0`